visualization/app/codeCharta/codeCharta.component.ts

```diff
@@ -42,20 +42,19 @@ export class CodeChartaController {
 		this.loadFileOrSample()
 	}
 
-	public loadFileOrSample() {
-		return this.urlUtils
-			.getFileDataFromQueryParam()
-			.then((data: NameDataPair[]) => {
-				if (data.length > 0) {
-					this.tryLoadingFiles(data)
-					this.setRenderStateFromUrl()
-				} else {
-					this.tryLoadingSampleFiles()
-				}
-			})
-			.catch(() => {
+	async loadFileOrSample() {
```

Comment:
Either set it to public here or let's remove the public type overall. Functions without a keyword there are implicit public. [This rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-member-accessibility.md) should do the job with the `no-public` property.

Reply:
Done